.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,41 +4,48 @@ about: Create a report about an issue or bug
 title: ''
 labels: bug
 assignees: ''
-
 ---
 
 ## Describe the bug
 <!-- A clear and concise description of what the bug is. -->
 
-## Steps to Reproduce
-<!--
+## Steps to reproduce the bug
+<!-- A step by step explanation on how to trigger the bug. -->
+
 Steps to reproduce the behavior:
+
 1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
--->
+2. Scroll down to '....'
+3. Click on '....'
 
```

Comment:
Here I took the steps out of the comment so users don't accidentially write inside the comment and are confused why the text doesn't show up.